pubnub-chat-impl/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ kotlin {
                 implementation(libs.kotlinx.datetime)
                 implementation(libs.kotlinx.serialization.core)
                 implementation(libs.kotlinx.atomicfu)
-                implementation(libs.lighthousegames.logging)
+                implementation("co.touchlab:kermit:2.0.4")
```

Comment:
please add this dependency to gradle/libs.versions.toml and use `libs.[...]` here (similar to other dependencies above it)